content/en/docs/armory-platform-matrix.md

```diff
@@ -360,23 +345,20 @@ Spinnaker requires an external storage provider for persisting app settings and
 
 | Provider              | Armory                 | Notes |
 | --------------------- | ---------------------- | ----- |
-| Azure Storage         | All supported versions |       |
 | Google Cloud Storage  | All supported versions |       |
 | Minio                 | All supported versions |       |
 | S3                    | All supported versions |       |
-| Oracle Object Storage | All supported versions |       |
 
-## Manifest templating
+Depending on the service, Spinnaker also uses either Redis, MySQL, or Postgres as a backing store. The following table lists the supported database and the Spinnaker service:
 
-[![Generally available](/images/ga.svg)]({{< ref "release-definitions#ga" >}}) ![OSS](/images/oss.svg) ![Armory](/images/armory.svg)
+| Database | DB version             | Armory                 | Spinnaker services                                  | Note                                                                                                                       |
+| -------- | ---------------------- | ---------------------- | --------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| Redis    | All supported versions | All supported versions | All Spinnaker services that require a backing store | The DB versions refer to external Redis instances. By default, Spinnaker deploys Redis internally to support its services. |
```

Comment:
I'd suggest a change.  
"Supported only for services that still require Redis for a feature (e.g. gate sessions).  Not supported as a core persistent storage engine." Add in the notes "Note that the internal redis is only supported for testing/poc purposes."